src/main/resources/application.yml

```diff
@@ -312,6 +314,7 @@ notifications:
           caveatWithdrawn: "c63bc430-9868-4f10-924b-408c7ac3cbd8"
           grantDelayed: "7045b381-a087-44fe-8bb5-e5a591cb8473"
           grantAwaitingDocumentation: "cc4aa69b-10fb-4a9c-9e0e-1851dfaa69e9"
+          sealedAndCertified: "16c9a4ce-162f-4ed1-8d70-be79616685ac"
```

Comment:
there is only english version template and it's hardcoded to get ENGLISH only.
`String templateId = notificationTemplates.getEmail().get(LanguagePreference.ENGLISH)`

prob we don't need to add it to the welsh section?